packages/components-organizations-registrar/src/pages/organizations/OrganizationCreate.jsx

```diff
@@ -242,18 +219,16 @@ const OrganizationCreate = () => {
     });
   };
 
-  const pageTitle = `Let’s get your organization registered on The Velocity Network™ ${config.chainName}.`;
-  const pageSubtitle =
-    'To start, fill in the form below to allow other Network participants to identify your organization.';
+  if (isLoading || isUserDataLoading) {
+    return <Loading sx={{ pt: '60px' }} />;
+  }
   return (
     <Box sx={sx.root}>
       <Stack sx={sx.content}>
-        <Typography variant="h1" mb={2} textAlign="center">
-          {pageTitle}
-        </Typography>
-        <Typography variant="pl" mb={6.5} textAlign="center">
-          {pageSubtitle}
-        </Typography>
+        <MockOrganization setInitialRecord={setInitialRecord} />
```

Comment:
I feel this should only be in the private repo as an extension to the core funcitonality in here